src/edu/csus/ecs/pc2/clics/API202306/CLICSVerionInfo.java

```diff
@@ -0,0 +1,48 @@
+// Copyright (C) 1989-2024 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.clics.API202306;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import edu.csus.ecs.pc2.VersionInfo;
+import edu.csus.ecs.pc2.services.core.JSONUtilities;
+
+/**
+ * CLICS Version Info.
+ * 
+ * @author Douglas A. Lane <pc2@ecs.csus.edu>
+ * @author John Buck
+ *
+ */
+public class CLICSVerionInfo {
```

Comment:
Name of class is misspelled (Verion -> Version).

Reply:
Did you see who wrote this module originally - haha ?

```
 * @author Douglas A. Lane <pc2@ecs.csus.edu>
 * @author John Buck

```
Doug certainly did this type of thing A LOT. :-) 
Fixd in nxt psh.